[2024-11-09T15:50:28.905+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_get_table.insert_postgres manual__2024-11-09T15:50:27.778925+00:00 [queued]>
[2024-11-09T15:50:28.911+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_get_table.insert_postgres manual__2024-11-09T15:50:27.778925+00:00 [queued]>
[2024-11-09T15:50:28.912+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-11-09T15:50:28.923+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): insert_postgres> on 2024-11-09 15:50:27.778925+00:00
[2024-11-09T15:50:28.926+0000] {standard_task_runner.py:60} INFO - Started process 1333 to run task
[2024-11-09T15:50:28.928+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'postgres_get_table', 'insert_postgres', 'manual__2024-11-09T15:50:27.778925+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/Code.py', '--cfg-path', '/tmp/tmp4f6yyhh8']
[2024-11-09T15:50:28.929+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask insert_postgres
[2024-11-09T15:50:28.965+0000] {task_command.py:423} INFO - Running <TaskInstance: postgres_get_table.insert_postgres manual__2024-11-09T15:50:27.778925+00:00 [running]> on host 2a2d16a2b1e8
[2024-11-09T15:50:29.024+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rob' AIRFLOW_CTX_DAG_ID='postgres_get_table' AIRFLOW_CTX_TASK_ID='insert_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-11-09T15:50:27.778925+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-09T15:50:27.778925+00:00'
[2024-11-09T15:50:29.030+0000] {base.py:83} INFO - Using connection ID 'Postgres' for task execution.
[2024-11-09T15:50:29.033+0000] {sql.py:457} INFO - Running statement: delete from public.tst2 where age in (19, 20, 24), parameters: None
[2024-11-09T15:50:29.034+0000] {sql.py:466} INFO - Rows affected: 1
[2024-11-09T15:50:29.036+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-11-09T15:50:29.042+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=postgres_get_table, task_id=insert_postgres, execution_date=20241109T155027, start_date=20241109T155028, end_date=20241109T155029
[2024-11-09T15:50:29.061+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-09T15:50:29.072+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
